packages/plugin-core/src/WorkspaceWatcher.ts

```diff
@@ -104,6 +104,13 @@ export class WorkspaceWatcher {
     this._windowWatcher = windowWatcher;
   }
 
+  // eslint-disable-next-line camelcase
+  __DO_NOT_USE_IN_PROD_exposePropsForTesting() {
+    return {
+      onFirstOpen: _.bind(this.onFirstOpen, this),
```

Comment:
Given that this is pseudo-public now (at least via testing), can you add a docstring to `onFirstOpen` on what the return boolean value represents?  It's not very clear what the ret value should be from a function called "onFirstOpen".  It should also help a reader to understand what the test is supposed to do.